modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/subscription/SubscriptionManagerImpl.kt

```diff
@@ -336,14 +336,27 @@ class SubscriptionManagerImpl @Inject constructor(
         freeTrialEligible = eligible
     }
 
-    override fun getDefaultSubscription(subscriptions: List<Subscription>): Subscription? {
-        val trialsIfPresent = subscriptions
+    override fun getDefaultSubscription(
+        subscriptions: List<Subscription>,
+        tier: Subscription.SubscriptionTier?,
+        frequency: SubscriptionFrequency?,
+    ): Subscription? {
+        val subscriptionTier = tier ?: Subscription.SubscriptionTier.PLUS
+        val subscriptionFrequency = frequency ?: SubscriptionFrequency.YEARLY
```

Comment:
This currently defaults to Plus tier and Yearly frequency. This logic is subject to change as users already configured to Plus should only be shown the Patron tier plans. The changes will be covered in a future PR.